wiki/Chat_Console/en.md

```diff
@@ -78,198 +76,36 @@ Commands list
-</ul></td>
-</tr>
-</tbody>
-</table>
+| Command        | Effect                                                                                         | Example           | BanchoBot response                                                               | 
+|------------------|------------------------------------------------------------------------------------------------|--------------------|----------------------------------------------------------------------------------|
+| /addfriend [user]   | Add [user] to your friends list.      | /addfriend Amigo | You are now friends with Amigo. |
+| /delfriend [user]   | Remove [user] from your friends list. | /delfriend Amigo | You are no longer friends with Amigo. |
+| /away [message] | Sets an away message (sent to users PMing you). Leave message blank to cancel. | /away I am John Smith. | You have been marked as being away: I am John Smith. When Amigo /msg John Where are you~?  BanchoBot: I am John Smith.|
+| /bb | Sends a message to bancho like rank [number] | /bb rank 1 | [15/11/12] Stats for Uan: Score: 47,323,299,680 (#1) Plays: 176293 (lv102) Accuracy: 98.95% |
+| /chat [user] | Open a new chat tab with specified user. | /chat Amigo | (Amigo tab is opened) | 
+| /clear | Clears the current chat buffer. | /clear | (Clears basically everything on the current tab) | 
+| /ignore [user][@chp] | Ignore all messages from specified user for this session. By adding an @ followed by the letters, c, h, and/or p, you may ignore them in chat, highlights, or PMs respectively. | /ignore Amigo@chp | BanchoBot: You will no longer hear Amigo {chat} {highlights} {PM} (Your chat console is set to: ignore any text written by Amigo [c], any possible text highlighting you by Amigo [h] any Private Message sent to you by Amigo [p]) | 
+| /j [channel] or /join [channel] | Joins the specified channel | /join #lobby | (#lobby tab is opened) | 
+| /p or /part | Leaves the current channel you're parting or leaving. | /part | n/a | 
+| /unignore [user] | Stop ignoring this user for this session. | /unignore Amigo | You may now hear Amigo. (Your chat console will allow any comment made by Amigo available to your chat console) | 
+| /me [action] | Perform a third-person action. | /me is at home | *John is at home | 
+| /msg [user] [msg] | Send a private message to [user]. | /msg Amigo I am sick at home. | (At Amigo tab) John: I am sick at home. | 
+| /np | Print to chat the current song you are listening to or playing. | /np | (If playing) *John is playing Peter Lambert - osu! tutorial [Gameplay basics] Game Modes Relax/Autopilot +Difficulty increasing mod -Difficulty decreasing mod/SpunOut #K. (If doing nothing, based on jukebox) *John is listening to Peter Lambert - osu! tutorial. (If spectating or Autoplay) *John is watching Peter Lambert - osu! tutorial [Gameplay basics] Game Modes Relax/Autopilot +Difficulty increasing mod -Difficulty decreasing mod/SpunOut #K Autoplay |
```

Comment:
don't know about this, but i think it should be xK instead of #K. It did in Game Modifiers.